website/src/db/index.js

```diff
@@ -0,0 +1,115 @@
+import Dexie from 'dexie';
+
+var db = new Dexie("AleoParameters");
+db.version(1).stores({
+    parameters: "name"
+});
+
+const FILES = [
+  // {
+  //   name: 'MintProver',
+  //   url: 'https://aleo-public.s3.us-west-2.amazonaws.com/testnet3/mint.prover.11fa6f2'
+  // },
+  // {
+  //   name: 'MintVerifier',
+  //   url: 'https://aleo-public.s3.us-west-2.amazonaws.com/testnet3/mint.verifier.e1ae0c8'
+  // },
+  {
+    name: 'TransferProver',
+    url: 'https://aleo-public.s3.us-west-2.amazonaws.com/testnet3/transfer.prover.837ad21'
```

Comment:
Right now, we're hosting these files. AleoHQ's CORS policy on the s3 bucket prevents this from working.